### DISCUSSION TITLE
[Feature] Categories and (unlimited loops of) Subcategories

### DISCUSSION THREAD

**Discussion by ghost** [nasirkhan/laravel-starter#558] [Ideas]
Hello Nasir!

In the demo, I found that it is not possible to create dependent subcategories. This can help everyone, if it is there. 

Then the contents can be better organised by having dependend and unlimited loops of subcategories.

The Laravel Starter template is really the best (although I have not used it due to some missing functions).

**Comment by ghost** [comment]
Hello Nasir!

So sad that this was not considered as a feature request. If one has one hunndred categories, which is not a surprise for many on websites having 1.000 posts, then having main categories gives an inherent and cluttered impression. Thus, having subcategories is an inevitable feature. I am sure many would need that but no one will request  here.

**Comment by nasirkhan** [comment]
I believe you are a long-time user of Laravel Starter. The feature you mentioned is not a must-have for most of the cases. But it is a good-to-have feature and not that complex to implement. For this reason, I did not spend time to implement this. 

But anyone may develop this feature and submit a PR, I would be very happy to accept that. 

One major update on the Category module is, that now it works independently and many-to-many relations can be created from any other module.

**Comment by ghost** [comment]
Hello my dear young friend Nasir!

`I believe you are a long-time user of Laravel Starter.`

I am not a user but have remained only as an observer. I had intended to use your project but after installing it I found you have developed this project on oneside with a lot of care and on the otherside delivered to this community a half cooked project.

Not all the users can or want to develop the project further. I did not want to modify your project. 

As these features were missing, I had to use some other project available on Github. 

And I am never ever going to use your project because it is defective on multilanguage feature.

However, I see it occassionally because I liked your work. But nothing more. No interest to use it.

Just like I had to find other projects for multilanguage and subcategories function, many users would do the same. But they will not write comments and will never take personal interest to discuss these things with you. Hence, you will remain under the illussion that no one needs it.

Sure, do not develop it. This is your choice. Still, what ever you have done and have offered it is a great work, atleast many can use it without multilanguage features and also without subcategories function.
